requirements/pip-tools.txt

```diff
@@ -2,25 +2,25 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --allow-unsafe --output-file=requirements/pip-tools.txt --strip-extras requirements/pip-tools.in
+#    pip-compile --allow-unsafe --output-file=/home/samccann/docsite/requirements/pip-tools.txt --strip-extras /home/samccann/docsite/requirements/pip-tools.in
```

Comment:
I know this is wrong but not sure what I did wrong locally so figured I'd just open the pr to get advice... What I did was run 'nox' and let it do all its things... and somehow this is the result.

Reply:
@oraNod @gotmax23 ^^

Reply:
@samccann I think we need to merge https://github.com/ansible/docsite/pull/239